### ISSUE TITLE
Finch on Ash

### ISSUE THREAD

**Issue by Shubhranshu153** [awslabs/automated-security-helper#67]
I am from finch team and have a few people reached out to me on how to use ash with finch on windows. So want to understand the ash arch better.

 From the readme it appears ash runs from inside the wsl environment. If it calls the finch command from inside the wsl i would assume it to fail as its not installed inside that environment rather its installed on windows environment. So i am not sure how ash achieves this communication, lets say with docker even. As docker also is installed in windows environment. Any pointers are helpful
 
 Thank You

**Comment by climbertjh2** [comment]
Yes, the documentation indicating the use of WSL for running ASH is now out-dated.  It was written **before** the latest over-haul of how ASH makes use of containers to run.

**Comment by KyleBooth** [comment]
I am trying to get ASH working on Windows using Docker. When I follow the above instructions, the image builds fine in Docker but I end up with the following error:
`exec /ash/ash: no such file or directory`

If I include the `-NoRun` tag, everything works fine.

**Comment by scrthq** [comment]
hey @KyleBooth - Thanks for working with me on this earlier today. I was able to reproduce the issue locally on my PC, CRLF line endings ended up in the repo and subsequently into the build container when building on Windows, resulting in syntax errors once `ash` was attempting to be invoked within the container. I've opened up PR #98 to address your issue and have confirmed I'm able to build and scan locally as well.

Note: This particular issue was not Finch specific and this issue is remaining open to update docs. We are keeping this issue open, but have updated steps due  to this issue that we will document and ultimately close this issue out as well.

**Comment by awsmadi** [comment]
Documentation for Windows installation (including the PowerShell `Invoke-ASH` guide from this thread) has been added to `docs/content/docs/installation-windows.md` in PR #312. Closing as documented.
